docs/bokeh/source/docs/first_steps/first_steps_1.rst

```diff
@@ -29,6 +29,12 @@ find links to both those resources.
 
 .. _first_steps_1_line_chart:
 
+Working with this guide
+-----------------------
+
+Bokeh is a Python library that does not have a graphical user interface (GUI). For the purpose of this guide, you can use simple Python scripts that you 
```

Comment:
Optimally, lines should not be longer than 88 characters. I'd add a line break between "the" and "purpose"!

Reply:
I'm a little worried that "this guide" might be construed as the entire user guide or docs. I might suggest something a little simpler:

> All the code in "First Steps" sections is plain Python code that can be be run as a standard Python script. 
> When run, these scripts will create HTML output that are visible in a web browser.

Reply:
One last suggestion, I am not sure an entire section heading is needed for this, perhaps it could just be a 
```
.. note::
```
at the end of the previous section?

Reply:
@bryevdv Did you mean at the end of the Installation details section?

Reply:
I think pretty much where it is now, just above the "Creating a simple line chart" section. You can find documentation about admonitions and specifically the note admonition here: https://docutils.sourceforge.io/docs/ref/rst/directives.html#specific-admonitions